docs/ui/interactions/fragments/web/chatbot.md

```diff
@@ -0,0 +1,15 @@
+ChatBot component is currently published on `ui-preview` tag for customer feedback.
+
+[![version](https://img.shields.io/npm/v/@aws-amplify/ui-react/ui-preview.svg)](https://www.npmjs.com/package/@aws-amplify/ui-react)
```

Comment:
This is React specific. We can maybe wrap this is a `docs-filter` for other frameworks.

Reply:
I'll wrap it

Reply:
Done!